omahelsinki/static/css/home.scss

```diff
@@ -0,0 +1,95 @@
+@import "./colors.scss";
+
+.omacontainer{
+    background: url(https://source.unsplash.com/collection/145103/1600x900);
+    background-repeat: no-repeat;
+    background-position: center;
+    background-size: cover;
+    height: 45vh;
+   color: $hel-white;
```

Comment:
The indentation is 3 spaces here. Maybe consider an SCSS linting tool for your editor?